src/asana/helpers.py

```diff
@@ -309,6 +309,31 @@ def urlreplace(matchobj: Match[str]) -> str:
     return re.sub(URL_REGEX, urlreplace, text)
 
 
+def get_linked_task_ids(pull_request: PullRequest) -> List[str]:
+    """
+    Extracts linked task ids from the body of the PR.
```

Comment:
The test is good for documentation, but adding a description here of the format of what we expect and what this is extracting would be helpful